### PR TITLE
chore(release): release script and initial changelog

### PR DESCRIPTION
**Pull request by lklimek**
## Issue being fixed or feature implemented

We need a way to easily generate a changelog

## What was done?

1. Added script, scripts/release/release.sh, that can do the release
2. The script uses [git-cliff](https://github.com/orhun/git-cliff) to generate changelog based on the git commit history

## How Has This Been Tested?

Created pull request https://github.com/dashevo/tenderdash/pull/251 that shows the output (to be updated when #250 is merged).

To be further tested during 0.7.0 release.

## Breaking Changes

none

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: scripts/release/pr_description.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+None
+
+## Breaking Changes
+<!--- Please describe any breaking changes your code introduces and verify that -->
+<!--- the title includes "!" following the conventional commit type (e.g. "feat!: ..."-->
+None
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+- [x] I have performed a self-review of my own code
```
Comment (shumkov) [inline scripts/release/pr_description.md:27]: I think the point of having this checkpoints is that you need always go through it make sure you did everything. So let's keep them unchecked.

**Comment by shumkov** [comment]
Does it allow to make dev and stable releases simultaneously?

**Review by shotonoff — APPROVED** [review]
🚀

File: scripts/release/pr_description.md (hunk 1 above)
Comment (lklimek) [reply scripts/release/pr_description.md:27]: This is just copied from [the Platform repo](https://github.com/dashevo/platform/blob/v0.22-dev/scripts/release/pr_description.md), but OK, I will update it.

**Pull request merged**